config/core/test-configs.yaml

```diff
@@ -193,6 +193,7 @@ test_plans:
     params:
       job_timeout: '10'
       kselftest_collections: "futex"
+      kselftest_skipfile: "skipfile-futex.yaml"
```

Comment:
Is this file actually defined anywhere?  It would most likely need a PR in the [test-definitions](https://github.com/kernelci/test-definitions) repository to add this file.

Reply:
@gctucker it's here: https://github.com/kernelci/test-definitions/pull/8